examples/complete-alb/main.tf

```diff
@@ -530,15 +456,14 @@ module "lambda_with_allowed_triggers" {
 
 module "lambda_without_allowed_triggers" {
   source  = "terraform-aws-modules/lambda/aws"
-  version = "~> 5.0"
+  version = "~> 3.0"
```

Comment:
```suggestion
  version = "~> 6.0"
```

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-alb/pull/305/commits/522c07009fa3cc79200886990f3ab8ba1ac76231